### ISSUE TITLE
Main page issues

### ISSUE THREAD

**Issue by pbln** [krushn-a/Isekai.io#7]
![image](https://github.com/user-attachments/assets/52864878-6b6e-4891-ba41-e6b45bf720e9)
Is this overlapping done intentionally or needs to be fixed , if needs to be fixed please assign this to me.

**Comment by dipandhali2021** [comment]
yes it seems there has been some issue of overlapping text, but it needs to be fixed across all devices for the website to be responsive. I am assigning this issue to you since you raised it. Please before raising PR test its responsiveness across some devices.
